.github/workflows/validate.yml

```diff
@@ -0,0 +1,32 @@
+name: ci
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: lts/*
```

Comment:
This will just select the latest LTS version, not all of them. For that one needs to use a matrix

ref: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-running-multiple-versions-of-nodejs

Personally, I would prefer a hard coded list:

```yaml
strategy:
  matrix:
    node: [10.12.0, 12.0.0, 14.0.0, 16.0.0]
```

Potentially adding in something like "latest" last, but I think it's way more likely to accidentally break compatibility with an older version, rather than a new version of Node.js making a breaking change that affect us.

Reply:
I first did this, but reverted to what's in https://github.com/standard/eslint-config-standard/blob/master/.github/workflows/ci.yml as @voxpelli requested in https://github.com/standard/eslint-config-standard-jsx/pull/41#discussion_r729324775

Will revert it back